src/init.ts

```diff
@@ -0,0 +1,9 @@
+import { AppRegistry } from 'react-native';
+
+async function initializeApplication() {
+  const { Application } = require('@/App');
+
```

Comment:
So essentially,r right here would be a place to setup stuff like segment for example?

Reply:
Exactly 👍 

Also if you have other ideas on how we could do this, lmk! But yeah that's the goal.